src/applications/disability-benefits/2346/components/Accessories.jsx

```diff
@@ -53,7 +53,7 @@ class Accessories extends Component {
         </p>
         <p>
           If you need accessories like domes, wax guards, cleaning supplies, or
-          desiccant, call the DLC Customer Service Station at{' '}
+          desiccant, call the DLC Customer Service Section at{' '}
```

Comment:
Should this be Service Section or Service Center?

Reply:
> Should this be Service Section or Service Center?

I remember them saying Section instead of Center. And their new design for the additional info component says Section.